vendor/revisions.json

```diff
@@ -1,14 +1,14 @@
 {
   "v16": [
     "16.4",
-    "6e9a4ff6249ac02b8175054b7b3f7dfb198be48b"
+    "e6203b5b66f8639461301a980e7a2fb1f9f7562f"
```

Comment:
@hlinnaka what's the status of this PR? I see that only 14-16 versions were bumped. Should we do the same for PG 17?

Reply:
yep, added that